docs/benchmarks/django-workload.md

```diff
@@ -0,0 +1,352 @@
+### Graphite Docker status/validation
+To check the status of Graphite container, execute the below script on the Graphite host machine
+```
+cloudsuite/benchmarks/django-workload/check_containers.sh <graphite-host-private-ip> 80 graphite
+```
+
+### Mandatory config
+The default config of the hopsoft/graphite-statsd docker container will likely
+cause your storage space to run out because of the amount of data being logged
+into statsd by the Django Workload. In order to solve this, please perform the
+steps below after starting the container. All commands should be run as root.
+
+Obtain a shell in the container:
+```
+sudo docker exec -it graphite bash
+cd opt/graphite/conf/
+```
+
+Add the following line to `blacklist.conf`:
+```
+^stats[^.]*\.benchmarkoutput\.
+```
+
+Edit the `carbon.conf` file to enable whitelisting. Search for the line
+containing `USE_WHITELIST`, uncomment it and set it to True:
+```
+USE_WHITELIST = True
+```
+
+Edit the retention policy in `storage-schemas.conf`:
+```
+[default_1min_for_1day]
+pattern = .*
+retentions = 10s:2h,1min:2d,10min:14d
+```
+
+Exit the docker container and restart it for the configurations to take effect:
+```
+exit
+docker stop graphite
+docker start graphite
+```
+
+Should the disk space fill up again, you can simply delete graphite’s database:
+```
+docker exec -it graphite bash
+rm –rf /opt/graphite/storage/whisper/*
+```
+
+## uWSGI configuration
+
+### uWSGI Docker status/validation
+To check the status of uWSGI container, execute the below script on the uWSGI host machine
+```
+cloudsuite/benchmarks/django-workload/check_containers.sh <uWSGI-host-private-ip> 8000 uwsgi
+```
+
+### Build uWSGI Image with custom Python build
+
+        $ ./build_uwsgi.sh [/absolute/path/to/cpython/install]
+
+### Debug
+
+#### uWSGI Logging
+Logging for uWSGI is turned off by default for benchmarking purposes. In order
+to turn it back on, comment out the `disable-logging` parameter in uwsgi.ini,
+like this:
+```
+#disable-logging = True
+```
+
+#### Django debugging
+If you get HTTP response codes different than 200, change the DEBUG parameter
+in cluster_settings.py:
+
+    DEBUG = True
+
+Then restart the uwsgi instance and go back to the page causing trouble. One of
+the most common problems that cause 400 codes is not having the correct host in
+the ALLOWED_HOSTS list in cluster_settings.py. If accessing the web server from
+a different machine, add the hostname/IP address you are using to access the
+server to ALLOWED_HOSTS.
+
+## Siege configuration
+Siege client will be used to stress test or benchmark the Python Django Workload.
+This will install Siege version 4.0.3rc3. There are no known version requirements
+for Siege at the moment, although older versions might not allow concurrency
+levels higher than 256, which can be insufficient to properly stress a certain
+system.
+
+### Configure Siege for your deployment
+Siege uses a urls.txt file to know where to direct the requests. This file can
```

Comment:
What about adding the path of urls.txt in the source code (e.g., cloudsuite/benchmarks/django-worload/siege/files/client) here?
It would make users find the file easily.

Reply:
The django workload builds a urls.txt while running the workload.
This section explains the same